### ISSUE TITLE
Full-screen in calls causes issues

### ISSUE THREAD

**Issue by francislavoie** [mattermost/desktop#2320]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: https://docs.mattermost.com/install/troubleshooting.html](https://docs.mattermost.com/install/troubleshooting.html).
- [x] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**

After full-screening and un-fullscreening a call (bottom-right button in the call), the app's menu bar (File/Edit/View) appears at the top, which pushes all of the rendered interface down however many pixels the menu bar takes up.

Also, while full-screened, the custom menu bar (three-dots menu + servers picker etc) is still at the top of the screen. I think it should not be there when full-screen because it cuts into the total viewable height of the call. It should be hidden when full-screen.

**Environment**
- Operating System:
- Mattermost Desktop App version: 5.1.1 
- Mattermost Server version: 7.2.0 

**Steps to reproduce**

- Start a call with someone
- Have them screen share
- Click Pop-out
- Click Full-screen 
- See that the custom menu bar is shown (point number 2)
- Close Full-screen
- See that the menu bar appeared (point number 1)

**Comment by cpoile** [comment]
Thanks @francislavoie for the report!

cc @streamer45 (it's linux)

**Comment by streamer45** [comment]
Thanks @francislavoie , I was able to reproduce this weird behaviour. I tried to go back a couple of versions and it seems the top bar has always been there but likely not many people noticed or triggered the fullscreen. However what changed from version 5.0.x to 5.1.x though is the view offset.

#### Screenshots

Before going fullscreen:

![image](https://user-images.githubusercontent.com/1832946/197967214-b348ced1-b6ab-4749-bc2b-65c8ce2da61a.png)

In fullscreen:

![image](https://user-images.githubusercontent.com/1832946/197967350-c3b46f79-9b74-45fa-8512-0740c7e277fc.png)

Exiting fullscreen:

![image](https://user-images.githubusercontent.com/1832946/197967476-67b52f4d-5c6b-498e-a0c1-3f8a669cbad7.png)

@devinbinnie Pinging you mostly to know if you have any idea on what could be causing this. This is nothing that we are doing directly from calls, it's just the native chrome video player going fullscreen which maybe is interfering with something else that was there already.

**Comment by devinbinnie** [comment]
> Thanks @francislavoie , I was able to reproduce this weird behaviour. I tried to go back a couple of versions and it seems the top bar has always been there but likely not many people noticed or triggered the fullscreen. However what changed from version 5.0.x to 5.1.x though is the view offset.
> 
> #### Screenshots
> Before going fullscreen:
> 
> ![image](https://user-images.githubusercontent.com/1832946/197967214-b348ced1-b6ab-4749-bc2b-65c8ce2da61a.png)
> 
> In fullscreen:
> 
> ![image](https://user-images.githubusercontent.com/1832946/197967350-c3b46f79-9b74-45fa-8512-0740c7e277fc.png)
> 
> Exiting fullscreen:
> 
> ![image](https://user-images.githubusercontent.com/1832946/197967476-67b52f4d-5c6b-498e-a0c1-3f8a669cbad7.png)
> 
> @devinbinnie Pinging you mostly to know if you have any idea on what could be causing this. This is nothing that we are doing directly from calls, it's just the native chrome video player going fullscreen which maybe is interfering with something else that was there already.

Sorry just to clarify, is the issue that clicking full-screen doesn't actually use the full screen?

Might be worth trying v5.2 to see if it's fixed in an Electron update: https://github.com/mattermost/desktop/releases/tag/v5.2.0-rc.3

Also which window manager are you using? That could have an effect here as well.

**Comment by streamer45** [comment]
> Sorry just to clarify, is the issue that clicking full-screen doesn't actually use the full screen?

I think there are two main issues here:

1. I'd expect the product/server switcher bar to hide when going full screen in the video element. Maybe that's something we need to configure somewhere?
2. When you exit full screen there's a weird offest to the view (this doesn't happen in 5.0.4 but it does in later versions). 

> Might be worth trying v5.2 to see if it's fixed in an Electron update 

I tried but it behaves the same. I'll try to reproduce on a less niche system as well, I use i3wm which clearly isn't the most supported but still part of this feels like app related somehow (at least the first point).

**Comment by francislavoie** [comment]
Yes, it's those two issues. Server bar appears while full-screen, and menu bar (file/edit/view) appears when exiting full-screen.

And to clarify, yes full-screen does work, it's just those couple details with the bars.

I'm using standard Gnome. It happened on both Ubuntu 20.10 and PopOS 22.04, both Gnome, but different versions of Gnome.
